proxmin/algorithms.py

```diff
@@ -70,25 +68,24 @@ def pgm(X0, prox_f, step_f, accelerated=False, relax=None, e_rel=1e-6, max_iter=
                 traceback.update_history(it+1, relax=relax)
 
         # test for fixed point convergence
-        if utils.l2sq(X - X_) <= e_rel**2*utils.l2sq(X):
+        converged = utils.l2sq(X - X_) <= e_rel**2*utils.l2sq(X)
+        if converged:
             break
 
     logger.info("Completed {0} iterations".format(it+1))
-    converged = True
-    if it+1 == max_iter:
+    if it+1 == max_iter and not converged:
```

Comment:
I think that this can be simplified to `if not converged`.

Reply:
👍